controllers/localization_controller.go

```diff
@@ -104,9 +107,13 @@ func (r *LocalizationReconciler) Reconcile(ctx context.Context, req ctrl.Request
 func (r *LocalizationReconciler) reconcile(ctx context.Context, obj *v1alpha1.Localization) (ctrl.Result, error) {
 	log := log.FromContext(ctx)
 	// get source snapshot
+
+	// TODO: If a resource source is defined, don't bother with the snapshot.
```

Comment:
Shouldn't snapshot take precedence over resource? My point of view is that it's cheaper to fetch a snapshot, most of the time because it is guaranteed to be in the local network.

Reply:
Yep.